config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -162,6 +162,7 @@ periodics:
       - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_secrets.yaml
       - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_statefulsets.yaml
       - --test-cmd-args=--testoverrides=./testing/experiments/use_simple_latency_query.yaml
+      - --test-cmd-args=--testoverrides=./testing/experiments/enable_prometheus_api_responsiveness.yaml
```

Comment:
This would have failed even in today`s run. I don't think we're ready for it.

Reply:
I had a look at last 3 runs of gce100 and kubemark500 and I haven't seen problems there. Can you please link the problematic run?

Reply:
I think Wojtek assumed (as I did) that it will affect 5K jobs as well.
Given that it changes only gce100 and kubemark500, which use simple prom query, I think this is fine. 

/lgtm
/approve

/hold
Please submit tomorrow morning to avoid accidentally breaking everyone for the night.

Reply:
Sorry - misread the change - I though it's for 5k-node test (which actually failed on Friday prometheus metrics).

Reply:
Yeah, I thought the same initially. @oxddr, could you change the PR title?

Reply:
Done,